content/02.main-text.md

```diff
@@ -192,6 +192,10 @@ This Sci-Hub Coverage Study preprint was the [most viewed](http://web.archive.or
 Hence, in Manubot's first year, two of the most popular preprints were written using its collaborative, open, and review-driven authoring process.
 
 Additional research studies in progress are being authored using Manubot, spanning the fields of [genomics](https://vsmalladi.github.io/tfsee-manuscript/), [climate science](https://openclimatedata.github.io/global-emissions/), and [data visualization](https://yt-project.github.io/yt-3.0-paper/).
+Papers with hundreds or thousands of authors are on the rise, such as the article describing the experiments and data that led to thd discovery of the Higgs Boson [@doi:10.1103/PhysRevLett.114.191803] (5000 authors) and the report of the Drosophila genome [@doi:10.1534/g3.114.015966] (1000 authors).
```

Comment:
One other typo: `thd discovery`

Reply:
Grr, thanks (was editing in Sublime, which is not my normal editor, and must have not enabled spell check). Fixed this one, and the one below, and split the text into its own paragraph.